sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -52,6 +52,8 @@ @implementation OOSkinViewController
 static NSString *kFrameChangeContext = @"frameChanged";
 static NSString *kViewChangeKey = @"frame";
 
+NSString *const OOOoyalaPlayerFullScreenChangedNotification = @"fullScreenChanged";
```

Comment:
sorry @wunnath  i disagree, this notification is specifically about the skin.

I would actually suggest a rename of "OOSkinViewControllerFullscreenChangedNotification"